src/lib/tls/tls_server.h

```diff
@@ -66,7 +66,8 @@ class BOTAN_PUBLIC_API(2,0) Server final : public Channel
       * tied to the session and a later renegotiation of the same
       * session can choose a new protocol.
       */
-      std::string next_protocol() const;
+      BOTAN_DEPRECATED("Use application_protocol() instead.")
```

Comment:
Why not directly remove it?

Reply:
We should just remove this - NPN support was removed long ago

Reply:
Removed. Along with `Server_Impl_12::m_choose_next_protocol` which was never actually set and marked as deprecated in the comments.